apisix/cli/ngx_tpl.lua

```diff
@@ -261,7 +271,7 @@ http {
 
     access_log {* http.access_log *} main buffer=16384 flush=3;
     {% end %}
-    open_file_cache  max=1000 inactive=60;
```

Comment:
open_file_cache is there: https://nginx.org/en/docs/http/ngx_http_core_module.html#open_file_cache
This directive is different from open_log_file_cache.

Reply:
Ack